intel_extension_for_transformers/transformers/modeling/modeling_auto.py

```diff
@@ -101,9 +101,14 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
                 torch_dtype = kwargs.pop("torch_dtype", torch.float32)
             if load_in_4bit:
                 if quantization_config is None:
-                    quantization_config = WeightOnlyQuantConfig(
-                        compute_dtype=torch_dtype, weight_dtype="nf4"
-                    )
+                    if use_llm_runtime: 
+                        quantization_config = WeightOnlyQuantConfig(
+                            compute_dtype="int8", weight_dtype="int4"

```

Comment:
compute_type should be bf16 also

Reply:
compute_type=bf16 can only run on spr machine